src/objects/aff/zcl_abapgit_object_common_aff.clas.abap

```diff
@@ -612,6 +623,24 @@ CLASS zcl_abapgit_object_common_aff IMPLEMENTATION.
       CATCH cx_root INTO lx_exception.
         zcx_abapgit_exception=>raise_with_text( lx_exception ).
     ENDTRY.
+  ENDMETHOD.
+
+  METHOD remove_abap_language_version.
+    DATA lv_pattern TYPE string.
+    DATA lv_string TYPE string.
+
+    lv_string = zcl_abapgit_convert=>xstring_to_string_utf8( iv_data = iv_json_as_xstring ).
 
+    lv_pattern = ',\s*"abapLanguageVersion"\s*:\s*"[^"]+"'.
+    " Replace the first occurence of "abapLanguageVersion" (in the header) only, so that
+    " further fields in the content can be still have the name "abapLanguageVersion"
```

Comment:
In my opinion the regex is too fragile. 
What happens if the abapLanguageVersion is the first field in the header, then the , will not match. 
What happens if the abapLanguageVersion is not contained in the header (the default is standard so it can be omitted). In this case we would match the next abapLanguageVersion field.

When you include the "header":{} into the regex, it should be more stable, what do you think?
`^"header"\s*:\s*\{.*?\s*("abapLanguageVersion"\s*:\s*"[^"]+").*?}`

Reply:
That doesn't handle the comma properly. The output must have a stable format and therefore also handle newlines (or there might be empty lines in the json output).

Something like this will be better:

```abap
data li_json type ref to zif_ajson.
li_json = zcl_ajson=>parse( json_string ).
li_json->delete( '/header/abapLanguageVersion` ).
json_string = li_json->stringyfy( iv_indent = 2 ).
```

Reply:
Using AJSON parser instead of regex, now.